src/WinGetUtil/ActivateThreadGlobals.cpp

```diff
@@ -0,0 +1,17 @@
+#include "pch.h"
+#include "Public/ThreadGlobals.h"
+
+namespace AppInstaller::ThreadLocalStorage
+{
+    ThreadGlobals* ThreadGlobals::ActivateThreadGlobals(AppInstaller::ThreadLocalStorage::ThreadGlobals* pThreadGlobals)
+    {
+        static AppInstaller::ThreadLocalStorage::ThreadGlobals* t_pThreadGlobals = nullptr;

```

Comment:
Similar approach as for tests, but no throwing, just an always global object (again, needs the same kind of setup helper object applied).